### ISSUE TITLE
Approach to ACT tests and accessibilty supported

### ISSUE THREAD

**Issue by alastc** [w3c/wcag3#771]
Given the updates to accessibility supported (where we will have a default "accessibility support set"), I think there is a change needed to how we frame ACT tests.

This also overlaps with the constant friction between "authors should just follow the standards", and "things need to work for disabled people in real life", mediated by the "accessibility supported" feature in WCAG.

In WCAG 2, the ACT tests have an assumed set of user-agents / AT that are tested against. If something doesn't work, it might be noted in the test.

In WCAG 3, we should have (and will define soon) a default "accessibility support set". Therefore we can carry on including examples which pass, regardless of whether they follow standards.

For example, in the [page-titled tests](https://github.com/w3c/wcag3/pull/696/changes#diff-65408a69cf2086445e07f918a6bb2af6bb9110d2b222024de3543b4a5a736871) it includes examples which are not valid HTML, but work in practice.

I'm raising this issue in case people want to argue that we should remove invalid examples.
